libcontainer/apparmor/apparmor_disabled.go

```diff
@@ -2,10 +2,19 @@
 
 package apparmor
 
+import (
+	"errors"
+)
+
+var ErrApparmorNotEnabled = errors.New("Apparmor: config provided but apparmor not supported")
```

Comment:
@hqhq 
Apologies for not following the right standard in error messages, will make the necessary change and do an commit back.
Thanks for your review.